src/sentry/static/sentry/app/views/settings/components/dataPrivacyRules/organizationRules.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import {css, ClassNames} from '@emotion/core';
+
+import space from 'app/styles/space';
+import {t} from 'app/locale';
+import Button from 'app/components/button';
+import {IconChevron} from 'app/icons';
+
+import RulesList from './rulesList';
+import {Rule} from './types';
+
+type Props = {
+  rules: Array<Rule>;
+};
+
+type State = {
+  isCollapsed: boolean;
+  contentHeight?: string;
+};
+
+class OrganizationRules extends React.Component<Props, State> {
+  state: State = {
+    isCollapsed: true,
+  };
+
+  componentDidUpdate() {
+    this.loadContentHeight();
+  }
+
+  rulesListRef = React.createRef<HTMLUListElement>();
+
+  loadContentHeight = () => {
+    if (!this.state.contentHeight) {
+      const contentHeight = this.rulesListRef.current?.offsetHeight;
+      if (contentHeight) {
+        this.setState({
+          contentHeight: `${contentHeight}px`,
+        });
+      }
+    }
+  };
+
+  handleToggleCollapsed = () => {
+    this.setState(prevState => ({
+      isCollapsed: !prevState.isCollapsed,
+    }));
+  };
+
+  render() {
+    const {rules} = this.props;
+    const {isCollapsed, contentHeight} = this.state;
+
+    if (rules.length === 0) {
+      return (
+        <Wrapper>
+          {t('There are no data privacy rules at the organization level')}
+        </Wrapper>
+      );
+    }
+    return (
+      <ClassNames>
+        {({css: classNamesCss}) => (
+          <Wrapper isCollapsed={isCollapsed} contentHeight={contentHeight}>
+            <Header onClick={this.handleToggleCollapsed}>
+              <div>{t('Organization Rules')}</div>
+              <Button
+                label={
+                  isCollapsed
+                    ? t('Expand Organization Rules')
+                    : t('Collapse Organization Rules')
+                }
+                icon={<IconChevron size="xs" direction={isCollapsed ? 'down' : 'up'} />}
+                size="xsmall"
+                labelClassName={classNamesCss`
```

Comment:
Not sure we should be special casing buttons. If the standard button is too big we could make a smaller standard size.